src/app.py

```diff
@@ -1,5 +1,7 @@
 import os
-
+import io
+import re
+import PyPDF2
```

Comment:
Could you please add this package to our `requirements.txt`? Our Dev Container will run pip install -r requirements.txt to install all packages defined under it. If you use a package without putting it under `requirements.txt` next time when you build a new Dev Container it will not have it installed anymore.

Reply:
For example I ran into this issue when using the Dev Container and trying to run the web with your code <img width="812" alt="Screen Shot 2023-10-26 at 6 35 11 PM" src="https://github.com/wangq131/4G06CapstoneProjectT5/assets/86222533/5fce376b-e782-4ce2-a0c8-676e6e727d60">